problems/2.functions.js

```diff
@@ -2,16 +2,19 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
+function logGreeting(name) {
+  console.log("Hello Avery!")
```

Comment:
Why Avery? Your function is taking in an argument called name. It's variable and my change, we want to greet whomever.